docs/src/content/docs/sdk.mdx

```diff
@@ -212,6 +212,25 @@ state.addEventListener('ultravoxExperimentalMessage', (msg) => {
 session.leaveCall();
 ```
 
+#### Debug Message:  Tool Call
+When the agent invokes a tool, the message contains the function, all arguments, and an invocation ID:
+```bash
+LLM response: Tool calls: [FunctionCall(name='createProfile', args='{"firstName":"Ron","lastName":"Burgandy","organization":"Fixie.ai","useCase":"creating a talking AI news reporter"}', invocation_id='call_D2qQVS8OQc998aMEw5PRa9cF')]
+```
+
+#### Debug Message:  Tool Call Result
+When the tool call completes, the message contains an array of messages. The first is the tool call. The second is the tool result. These messages are also available via [List Call Messages](../api/calls/#list-call-messages)
```

Comment:
This isn't actually quite right.  The model can invoke multiple tools at once.  This message will have all the calls followed by all the results.  (But I also think this is more information than necessary.)

Reply:
ok i am tweaking a bit to be more correct